dir.props

```diff
@@ -14,6 +14,7 @@
     <PortableBuild Condition="'$(PortableBuild)' == ''">false</PortableBuild>
     <UseSystemLibraries Condition="'$(UseSystemLibraries)' == '' AND '$(PortableBuild)' != 'true'">true</UseSystemLibraries>
     <UseStableVersions Condition="'$(UseStableVersions)' == ''">true</UseStableVersions>
+    <SkipPortableRuntimeBuild Condition="'(SkipPortableRuntimeBuild)' == ''">true</SkipPortableRuntimeBuild>
```

Comment:
Can we instead change the variable to `PortableRuntimeBuild` to avoid a double negative build arg and use the slightly more intuitive `false` default?